CMakeLists.txt

```diff
@@ -22,15 +22,20 @@ set(CMAKE_SKIP_RPATH TRUE)
 set(MCM_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(MP_DIR ${CMAKE_CURRENT_SOURCE_DIR}/media-proxy)
 set(SDK_DIR ${CMAKE_CURRENT_SOURCE_DIR}/sdk)
+set(TESTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests/unit)
 set(SDK_INCLUDE_DIR ${SDK_DIR}/include)
 
 # Setup output folder
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
+enable_testing()
+
 add_subdirectory(${SDK_DIR})
 add_subdirectory(${MP_DIR})
+add_subdirectory(${TESTS_DIR})
+
+target_include_directories(media_proxy_lib PUBLIC ${SDK_INCLUDE_DIR})
+target_link_libraries(media_proxy_lib PUBLIC mcm_dp)
```

Comment:
Great catch and idea for renaming component.
What would you say for fitting there also the `sdk` acronym?  Like `media_proxy_sdk_lib` or any other variation so that name will become fully descriptive?

Reply:
My intention was not to rename the component. I created the media_proxy_lib just to be able to link the media_proxy code to the unit tests. Before and after my changes, only the media_proxy executable is built, the media_proxy shared library is not created, so I'm not sure if the product of the compilation process can be called ‘sdk’.